QuizApp/App/Common/Coordinator.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+final class Coordinator: CoordinatorProtocol {
+
+    private var navigationController: UINavigationController?
```

Comment:
Initiate NavCon in SceneDelegate and assign it to window. Pass it to Coordinator via init.